### DISCUSSION TITLE
Consolidation and drainage process

### DISCUSSION THREAD

**Discussion by PEI0214** [idaholab/moose#27035] [Q&A Modules: Porous Flow]
Hi，
When I simulated the consolidation of soil in porous flow, the loading process was set as an undrained condition, and when the confining pressure was increased to the expected value, the pore pressure remained unchanged when the drainage was opened.  This is different from the expected gradual dissipation of pore pressure.Why?
However, when the surrounding pressure remains unchanged and the axial stress continues to increase, the pore pressure will quickly dissipate.

Thanks for your help.

**Comment by GiudGiud** [comment]
@cpgr
